projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -161,11 +161,9 @@ protected function inject_javascript_options() {
 
 		$options = array(
 			'overlayOptions'        => array(
-				'colorTheme'      => get_option( $prefix . 'color_theme', 'light' ),
```

Comment:
As we discussed, I'm currently trying to find out how many users have dark mode enabled. Might end up leaving this in and adding new styles for it in a followup PR 👍

Reply:
🤙

Reply:
@jsnmoon haven't been able to get stats yet, but my gut feeling is that we should leave this in. Shall we just comment the related code rather than removing it altogether, and aim to add styles in a followup?

Reply:
I think that's a better way to go, honestly. I'll work on restoring this (and maybe slap together some simple color styles for dark mode).